CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## 0.3.0
```

Comment:
went with a minor bump because (1) we're still at a `0.` release so _no rules_! and (2) this fixes core functionality for a number of significant usages